--- wormux-0.7.9/data/wormux.desktop~	2007-01-28 15:06:10.000000000 +0100
+++ wormux-0.7.9/data/wormux.desktop	2007-02-11 14:06:58.000000000 +0100
@@ -2,7 +2,7 @@
 Version=1.0
 Encoding=UTF-8
 Type=Application
-Icon=wormux_128x128.png
+Icon=wormux.xpm
 Exec=wormux
 Terminal=false
 StartupNotify=false
@@ -12,9 +12,10 @@
 Comment=A 2D game where funny characters launch funny weapons
 Comment[de]=Ein 2D Spiel mit lustigen Figuren und Waffen
 Comment[es]=Un juego en 2D donde extraños personajes se destripan con alegría
-Comment[fr_FR]=Un jeu en 2D où de drôles de personnages s'étripent joyeusement
+Comment[fr]=Un jeu en 2D où de drôles de personnages s'étripent joyeusement
 Comment[it]=Un gioco in 2D dove dei simpatici personaggi si sparano allegramente
-Comment[pl_PL]=Gra 2D, w ktorej smieszne postacie uzywaja smiesznych typow broni
-Comment[ro_RO]=Un joc în 2D în care personaje amuzante lansează arme amuzante
+Comment[pl]=Gra 2D, w której śmieszne postacie używają śmiesznych typów broni
+Comment[ro]=Un joc în 2D în care personaje amuzante lansează arme amuzante
 
-Categories=Application;Game;ArcadeGame
+Categories=Game;ArcadeGame
+# vi: encoding=utf-8
