--- wormux-0.7.9/src/Makefile.am.orig	2007-02-09 01:27:53.169072787 +0100
+++ wormux-0.7.9/src/Makefile.am	2007-02-09 01:28:00.297479010 +0100
@@ -176,7 +176,7 @@
 	weapon/blowtorch.cpp weapon/blowtorch.h \
 	weapon/syringe.cpp weapon/syringe.h
 
-AM_CXXFLAGS = -DINSTALL_DATADIR=\"$(pkgdatadir)\" -DINSTALL_LOCALEDIR=\"$(localedir)\" -DFONT_FILE=\"$(fontfile)\" -DPACKAGE_VERSION=\"$(PACKAGE_VERSION)\" -Werror
+AM_CXXFLAGS = -DINSTALL_DATADIR=\"$(pkgdatadir)\" -DINSTALL_LOCALEDIR=\"$(localedir)\" -DFONT_FILE=\"$(fontfile)\" -DPACKAGE_VERSION=\"$(PACKAGE_VERSION)\"
 
 
 if WIN32
